docs/BEST_PRACTICES.md

```diff
@@ -134,6 +134,180 @@ sealed class Foo {
 #### Tip
 To display a helpful viewer for TODO and FIXME tags in Android Studio, go to Navigate -> Tool Windows -> TODO
 
+### Jetpack Compose
+#### Compose Module
+Compose UI code is being developed in a separate module to decrease compile time for Compose previews.  This also enforces separation of concerns to avoid references to viewmodels and other non state related code from being coupled to UI code.
+
+#### Compose Screens
+##### State
+Composable screens should be driven by a state object and lambas.  This state class is defined at the screen UI level in the compose module.  It can be used to drive previews of the screen.
```

Comment:
@BottleRocket-Colin Couple of things:

1. typo in lambda (corrected in snippet below)
2. I think it might be helpful to also mention this class will prevent you from attempting to reference the ViewModel directly and helps create fake data for previews easily (I've modified the last line in the snippet below)

```suggestion
Composable screens should be driven by a state object and lambdas.  This state class is defined at the screen UI level in the compose module.  It can be used to drive previews of the screen easily with fake data, which wouldn't be the case if viewmodels were directly referenced.
```